contrib/debian/postrm

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# postrm script for aziot-identity-service
+#
+# see: dh_installdeb(1)
+
+set -e
+
+case "$1" in
+	purge)
+		systemctl daemon-reload
+
+		# Delete directories used by aziot-identity-service.
+		# Home directories are not deleted here; they will be deleted with their user.
+		if [ -d /etc/aziot ]; then
+			services=("identityd" "certd" "keyd" "tpmd")
+
+			for service in "${services[@]}"; do
+				rm -rf "/etc/aziot/${service}"
+			done
+
+			if [ -z "$(ls -A /etc/aziot)" ]; then
+				rm -rf /etc/aziot
+			fi
+		fi
+
+		directories=("/run/aziot" "/var/secrets/aziot")
```

Comment:
`/var/secrets/aziot` doesn't belong to us. It contains user files.

Reply:
Fixed.